schema@v1.1.1/dataset.json

```diff
@@ -7,8 +7,12 @@
       "$ref": "./schema@v1.1.1#/definitions/basicProperties"
     }
   ],
-  "required": ["tables", "version", "status"],
+  "required": ["tables", "version", "status", "path"],
   "properties": {
+    "path": {
+      "type": "string",
+      "format": "uri"
```

Comment:
Vraag: In de aangepaste schema's is dit overal gewoon een string. Waarom is het  als  `format` `uri` gedefinieerd? Zou het bijv. wel een path kunnen zijn?

Reply:
path kan bijvoorbeeld `haalcentraal/brp` zijn.

Reply:
Sorry, ik had het commentaar van @jjmurre niet gezien. Maar een uri moet volgens JSON Schema wel een volledige URI zijn, beginnend met een *scheme* en een dubbele punt. De paden die we gebruiken zijn geen URI's.

Reply:
`"format": "uri-reference"` is misschien beter (https://json-schema.org/understanding-json-schema/reference/string.html#resource-identifiers), maar wat mij betreft kunnen we het ook leeg laten en alleen het type opgeven.